components/compliance-service/compliance.go

```diff
@@ -517,7 +491,13 @@ func Serve(conf config.Compliance, grpcBinding string) error {
 		return err
 	}
 	SERVICE_STATE = serviceStateStarting
```

Comment:
I _think_ we can probably get rid of SERVICE_STATE as well, but the startup is complicated enough that I need to put some more thought into it.